.github/workflows/ci.yml

```diff
@@ -214,6 +214,16 @@ jobs:
         with:
           mdbook-version: '0.4.17'
 
+  mdbook-lint:
+    needs: cancel-previous-runs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - run: npm install -g markdownlint-cli && markdownlint --config .markdownlintrc **/*.md docs/src/ 
```

Comment:
I would prefer if an exact version of markdownlint-cli were installed. I've had too many nightmares of JS dependencies getting bitrot and suddenly stop working 😂

Reply:
That's fair. Updates pushed